src/SFML/Window/OSX/SFOpenGLView.hpp

```diff
@@ -68,8 +68,7 @@ class WindowImplCocoa;
 /// the cursor (that was disconnected from the system).
 ///
 ////////////////////////////////////////////////////////////
-@interface SFOpenGLView : NSOpenGLView
-{
+@interface SFOpenGLView : NSOpenGLView {
```

Comment:
This got reformatted after changing the extension presumably because now clang-format knows that it's an Objective-C++ file but I'm not entirely certain. Same goes for SFViewController and SFWindowController.

Reply:
https://github.com/SFML/SFML/blob/9855552f6431016f1953e36b2f058fbd239ed725/.clang-format#L171

That's what `AfterObjCDeclaration` controls. It just bugs out when the file extension isn't `.h`. You can set this to `false` and you'll see that another .h Objective-C file changes how it formats this brace after `@interface` so I know this is the right setting to use in this situation.